languages.toml

```diff
@@ -396,6 +397,21 @@ indent = { tab-width = 2, unit = "  " }
 name = "elixir"
 source = { git = "https://github.com/elixir-lang/tree-sitter-elixir", rev = "02a6f7fd4be28dd94ee4dd2ca19cb777053ea74e" }
 
+[[language]]
+name = "fennel"
+scope = "source.fennel"
+file-types = ["fnl", "fnlm"]
+shebangs = ["fennel"]
+comment-token = ";"
+language-servers = ["fennel-ls"]
+formatter = { command = "fnlfmt", args = ["-"]}
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "fennel"
+source.git = "https://github.com/alexmozaidze/tree-sitter-fennel"
+source.rev = "ea4a536bca8997e30b22709f210f44f97e75bf7d"
```

Comment:
It looks like this revision is fairly old. Is there any reason not to use the latest commit?